src/OpenLoco/src/World/CompanyAi.cpp

```diff
@@ -1572,18 +1577,78 @@ namespace OpenLoco
     }
 
     // 0x00487BA3
-    static bool sub_487BA3(AiThought& thought)
+    // Sells one at a time
+    // returns true when there are no vehicles left to sell
+    static bool sellAiThoughtVehicle(AiThought& thought)
```

Comment:
This name is a bit confusing, I think. This represents the AI thinking about selling a vehicle, right? It might be clearer to lead with that, e.g. `aiThinkSellVehicle`?

Reply:
Its selling a vehicle that is owned by the AiThought. Perhaps calling them `AiThought`s wasn't the best idea but couldn't think of anything better.
If it was selling all vehicles I would have called it `aiSellAllVehicles`

Reply:
Perhaps `sellVehicleByAiThought` then? `By...` referring to the parameter.